src/main/java/io/khasang/jrepetitor/model/CreateTable.java

```diff
@@ -37,19 +37,5 @@ public String createTableStatus() {
         } catch (Exception e) {
             return "Table creation failed " + e;
```

Comment:
you have task remove something from app?

Reply:
done (local)